src/components/TabSelector/TabSelector.tsx

```diff
@@ -74,6 +74,7 @@ function TabSelector({
     const styles = useThemeStyles();
     const defaultAffectedAnimatedTabs = useMemo(() => Array.from({length: state.routes.length}, (v, i) => i), [state.routes.length]);
     const [affectedAnimatedTabs, setAffectedAnimatedTabs] = useState(defaultAffectedAnimatedTabs);
+    const [view, setView] = useState<View | null>(null);
```

Comment:
Tests well!
I wonder if it's possible to simplify this so we don't store a whole Component reference in state.
Would simply passing a reference to `TabSelectorItem` work?